src/Symfony/Component/HttpFoundation/ParameterBag.php

```diff
@@ -188,7 +201,7 @@ public function getEnum(string $key, string $class, ?\BackedEnum $default = null
         try {
             return $class::from($value);
         } catch (\ValueError|\TypeError $e) {
-            throw new UnexpectedValueException(\sprintf('Parameter "%s" cannot be converted to enum: %s.', $key, $e->getMessage()), $e->getCode(), $e);
+            throw new UnexpectedValueException(\sprintf('Parameter "%s" cannot be converted to enum: "%s".', $key, $e->getMessage()), $e->getCode(), $e);
```

Comment:
false-positive, which we can fix this way:
```suggestion
            throw new UnexpectedValueException(\sprintf('Parameter "%s" cannot be converted to enum: ', $key).$e->getMessage().'.'), $e->getCode(), $e);
```

Reply:
Done